prog/vnet/load_balancer_health_probes.rb

```diff
@@ -8,18 +8,24 @@ def vm
   end
 
   label def health_probe
-    response_code = begin
+    address_ipv4 = load_balancer.ipv4_enabled? ? vm.nics.first.private_ipv4.network : nil
+    address_ipv6 = load_balancer.ipv6_enabled? ? vm.ephemeral_net6.nth(2).to_s : nil
```

Comment:
I would either keep them both as String or both as NetAddr objects. Probably the latter:
```
    address_ipv4 = load_balancer.ipv4_enabled? ? vm.nics.first.private_ipv4.network : nil
    address_ipv6 = load_balancer.ipv6_enabled? ? vm.ephemeral_net6.nth(2) : nil
```

Then you can can later check if `address.version` is `4` or `6`, instead of checking whether the string includes a colon.